app/assets/stylesheets/components/_youtube-player.scss

```diff
@@ -39,3 +39,20 @@
     border-radius: 100%;
   }
 }
+
+.govuk-details__summary {
```

Comment:
Nitpick could we please add
```
.govuk-details__summary {
  color: govuk-colour('white');
  &:hover {
    color: govuk-colour('white');
  }
  &:hover:focus,
  &:hover:focus .govuk-details__summary-text {
    color: govuk-colour('black');
  }
}
```

This should ensure that the text and arrow are legible when in focus. With the current styling, if the element is in focus the text turns white on a yellow background. This is a WCAG colour contrast fail.

The solution above is not 100% ideal but it should do in order to make it look decent for now 😕

Reply:
Thanks Dana